typescript/infra/scripts/funding/fund-relayers-from-deployer.ts

```diff
@@ -158,4 +178,49 @@ function getRelayerKeysFromSerializedAddressFile(path: string): AgentKey[] {
     .filter((key: AgentKey) => key.role === KEY_ROLE_ENUM.Relayer);
 }
 
+function submitFunderBalanceMetrics(balances: FunderBalance[]) {
+  const gatewayAddr = process.env['PROMETHEUS_PUSH_GATEWAY'];
+  if (!gatewayAddr) {
+    console.warn(
+      'Prometheus push gateway address was not defined; not publishing metrics.',
+    );
+    return;
+  }
+
+  const register = new Registry();
+  // TODO: get actual push gateway address
+  const gateway = new Pushgateway(gatewayAddr, [], register);
+  const gauge = new Gauge({
+    name: 'abacus_relayer_funder_balance',
```

Comment:
Wondering if you considered using the same metric name / labels as we already use in the agents here https://github.com/abacus-network/abacus-monorepo/blob/main/rust/abacus-base/src/metrics/provider.rs#L41. I see downsides to this but also potential upsides, so I'm happy with whatever you choose

Reply:
Main reason why my gut went to doing it as a different metrics was because this one may be kinda sporadic or have long periods without being reported, so I was thinking we will need to treat it differently but in hindsight this is untrue because the push-gateway continues to present it and the balance won't change if the script won't run.

So yes, good idea!